machina/templates/machina/partials/form_field.html

```diff
@@ -2,10 +2,16 @@
 {% load widget_tweaks %}
 
 <div id="div_id_{{ field.html_name }}" class="form-group{% if field.errors %} has-error{% endif %}">
+  {% if not field|widget_type == 'checkboxinput' %}
```

Comment:
Then I think a new partial for the checkbox is best. We have to use it at least two places now.